src/Microsoft.Framework.Configuration.Json/JsonConfigurationSource.cs

```diff
@@ -78,7 +79,19 @@ public override void Load()
         internal void Load(Stream stream)
         {
             JsonConfigurationFileParser parser = new JsonConfigurationFileParser();
-            Data = parser.Parse(stream);
+            try
+            {
+                Data = parser.Parse(stream);
+            }
+            catch(JsonReaderException e)
+            {
+                var splitErrorMessageArray = e.Message.Split(new string[] { e.Path }, StringSplitOptions.None);
```

Comment:
Can you add a comment of what you expect for e.Message and e.Path? It's impossible to know what this code does without knowing the inputs.